### PR TITLE
Add blog post for PHP Tour

### PR DESCRIPTION
**Pull request by willdurand**

### CODE REVIEW

**Comment by lunika** [comment]
:+1:

**Comment by krichprollsch** [comment]
:+1:

File: _posts/2016-01-16-phptour-clermont-fd-2016.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+débarquera les **23 et 24 mai 2016** au Polydôme de Clermont-Ferrand, le centre
+de congrès à proximité du centre-ville et aux abords du stade de rugby Marcel
+Michelin. La prochaine étape du cycle de conférences itinérant **PHP Tour**,
+réunissant toutes les communautés PHP, professionnelles et open-source, se
+déroulera à l'intérieur de nos volcans !
+
+Le PHP Tour, ce sont **deux jours** de conférences pleinières de 20 ou 40
+minutes, des _lightning talks_ (c'est-à-dire des conférences très courtes, plus
+courtes que nos [API Hours](/api-hours/)), mais également des ateliers. Cet
+événement attire chaque année quelques centaines de personnes de tout horizon,
```
Comment (jmaupetit) [inline _posts/2016-01-16-phptour-clermont-fd-2016.md]: "de tous horizons" non ?

Comment (willdurand) [inline _posts/2016-01-16-phptour-clermont-fd-2016.md]: Ouais.

**Pull request merged**